client/src/components/infoDrawer/infoFormat.js

```diff
@@ -0,0 +1,194 @@
+import { H3, H1, UL, Classes } from "@blueprintjs/core";
+import React from "react";
+
+const renderContributors = (contributors, affiliations, skeleton) => {
+  // eslint-disable-next-line no-constant-condition --  Temp removed contributor section to avoid publishing PII
+  if (contributors?.length > 0 && false)
+    return (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>Contributors</H3>
+        <p className={skeleton ? Classes.SKELETON : null}>
+          {contributors.map((contributor) => {
+            const { email, name, institution } = contributor;
+
+            return (
+              <span key={name}>
+                {name}
+                {email && `(${email})`}
+                <sup>{affiliations.indexOf(institution) + 1}</sup>
+              </span>
+            );
+          })}
+        </p>
+        {renderAffiliations(affiliations, skeleton)}
+      </>
+    );
+  return null;
+};
+
+const buildAffiliations = (contributors = []) => {
```

Comment:
Just curious to see if you like the `new Set()` approach here, since I feel it's less code and easier to read 😆 

```ts
const buildAffiliations = (contributors) => {
  const affiliations = contributors.map(
    (contributor) => contributor.institution
  );

  return Array.from(new Set(affiliations));
};
```

Reply:
I initially used the set, but went in favor of the explicit "only push unique values" as I thought it was more explicit and transparent on the intended functionality rather than implicitly in the Set constructor.  I'm not dead set on this method, so let me know if you think I should switch :)

Reply:
Ahh got it! Yeah either way is fine, I think this is more subjective at this point 😆 LGTM!